Backend variables (#430)
avenga/couper: docs/REFERENCE.md

- `id_token_claims`: a map of claims from the ID token
- `userinfo`: a map of claims retrieved from the userinfo endpoint

> ### `backend_request`
> 
> `backend_request` is a list of request variables like `backend_requests.<label>`
> (see [backend_requests](#backend_requests) below). The `backend_request` is only
> available in a [Backend Block](#backend-block) and represents the request of this
> block.
> 
> ### `backend_requests`
> 
> `backend_requests.<label>` is a list of all backend requests, and their variables.

Reviewer: ```suggestion
### `backend_request`

`backend_request` holds information about the current backend request. It is only
available in a [Backend Block](#backend-block), and has the same attributes as a backend request in `backend_requests.<label>` (see [backend_requests](#backend_requests) below).

### `backend_requests`

`backend_requests` is an object with all backend requests and their attributes.
```

| `port`                           | integer         | Port of the backend request URL                                                                                                                                                                                                                                                      | `443`                                       |
| `path`                           | string          | Backend request URL path                                                                                                                                                                                                                                                             | `/path/to`                                  |

> ### `backend_response`
> 
> `backend_response` is a list of responses variables like `backend_responses.<label>`
> (see [backend_responses](#backend_responses) below). The `backend_response` is only
> available in a [Backend Block](#backend-block) and represents the response of this
> block.
> 
> ### `backend_responses`
> 
> `backend_responses.<label>` is a list of all backend responses, and their variables. Same behaviour as for `backend_requests`.

Reviewer: ```suggestion
### `backend_response`

`backend_response` represents the current backend response.  It is only
available in a [Backend Block](#backend-block), and has the same attributes as a backend response in `backend_responses.<label>` (see [backend_responses](#backend_responses) below).

### `backend_responses`

`backend_responses` is an object with all backend responses and their attributes.
```
